src/test/java/learning/SetCollectionTest.java

```diff
@@ -0,0 +1,56 @@
+package learning;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetCollectionTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인한다.")
+    void testSize() {
+        //assert
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
다음과 같이도 작성해볼 수 있습니다. 미션을 진행하시면서 다양한 API를 활용해보시는 걸 추천드려요.
```suggestion
        assertThat(numbers).hasSize(3);
```